mempool/cache.go

```diff
@@ -18,14 +18,14 @@ type TxCache interface {
 
 	// Push adds the given raw transaction to the cache and returns true if it was
 	// newly added. Otherwise, it returns false.
```

Comment:
[blocking]

```suggestion
	// Push adds the given tx key to the cache and returns true if it was
	// newly added. Otherwise, it returns false.
```